guides/micronaut-http-client/groovy/src/main/groovy/example/micronaut/GithubApiClient.groovy

```diff
@@ -0,0 +1,14 @@
+package example.micronaut
+
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.Header
+import io.micronaut.http.client.annotation.Client
+import io.reactivex.Flowable
+
+@Client(GithubConfiguration.GITHUB_API_URL) // <1>
+@Header(name = "User-Agent", value = "Micronaut HTTP Client") // <2>
```

Comment:
I'm not sure how important it is, but they recommend adding `Accept: application/vnd.github.v3+json`